src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -330,9 +330,10 @@ def project_or_package_link(opts)
     defaults = { package: nil, rev: nil, short: false, trim_to: 40 }
     opts = defaults.merge(opts)
 
-    CacheLine.fetch(['project_or_package_link', opts], project: opts[:project], package: opts[:package]) do
-      # only care for database entries
-      prj = Project.where(name: opts[:project]).select(:id, :name).first
+    # only care for database entries
+    prj = Project.where(name: opts[:project]).select(:id, :name).first
```

Comment:
Autsch. I missed that one:/ Added the updated_at attribute now.

> evanrolfe 17 hours ago
> prj = Project.find_by(name: opts[:project])

Last time I tried, rubocop wanted me to use .where().first